script/openqa-webui-daemon

```diff
@@ -1,3 +1,38 @@
-#!/bin/sh -e
-# Our API commands are very expensive, so the default timeouts are too tight
-exec "$(dirname "$0")"/openqa prefork -m production --proxy -i 100 -H 900 -w 30 -c 1 -G 800 "$@"
+#!/bin/bash
+set -e
+
+pid=
+pid_idx=0
+pid_dir=${OPENQA_BASEDIR:-/var/lib}/openqa/webui
+openqa_args=("$@")
+openqa_dir=$(dirname "$0")
+
+function start_service {
+    # keep track of the previous and next PID
+    pid_last=$pid
+    pid_file=$pid_dir/prefork-$pid_idx.pid
```

Comment:
this would produce `prefork-_idx.pid` on first invocation, or do I miss something?

Reply:
No, all the variables are default initialized.

Reply:
Initialized, yes but empty: https://github.com/os-autoinst/openQA/pull/5820/files/a1f44e49621ea7f8bae63853c11400fa079069a3#diff-88ceba577fe4e4d895b021004b958cac9c30cb780fc1f443d6bcdc54f362377eR4 - It just contains a value the second the function is called

Reply:
But this is fine. `pid_last=$pid` makes sense even if `pid` is empty because of the check `[[ $pid_last ]] && …` below. Note that I also tested this and the PID files always had the expected name and `kill` was never executed accidentally or wrong arguments (which would lead to a notable failure via `set -e`).

Reply:
yes you're right. I actually mis-clicked and was more concerned about the pid_file above but it uses `pid_idx` and not just `pid` - that is why I usually like ${} a little bit more. Anyhow, no problem then.